### DISCUSSION TITLE
Clarification on Using Preline Pro Components as Inspiration for Open Source Projects

### DISCUSSION THREAD

**Discussion by TedKrj** [htmlstreamofficial/preline#482] [General]
Hi Preline team,

I’m working on an open-source project using Svelte, and I’m considering incorporating Preline UI. I understand that the Pro version of Preline cannot be used in open-source projects, but I’d like some clarification regarding the following:

Is it permissible to take inspiration from the layout of Preline Pro components, without directly copying the HTML or class names, and implement a similar design using the free Preline components?

For example, if I like the layout of a review page or the list group with switch in Preline Pro, can I recreate a similar page using only the free version components, while maintaining a different implementation?

I want to ensure I’m respecting the licensing terms while still drawing design inspiration from Pro components.

Thank you for your guidance!

<img width="405" alt="image" src="https://github.com/user-attachments/assets/9f75c972-79fc-4d0d-8c28-6ff9eeb5b3ab">

<img width="707" alt="image" src="https://github.com/user-attachments/assets/c717e805-6804-488f-8853-f5b263e6a9c0">

**Comment by jahaganiev** [comment]
Hey,

The Pro version is designed for commercial use and should not be replicated in any form.

Hope that helps!
